templates/taxi/index.html

```diff
@@ -9,5 +9,6 @@ <h2>Dynamic content</h2>
     <li><strong>Cars:</strong> {{ num_cars }}</li>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
+    <li>Num visits: {{ num_visits }}</li>
```

Comment:
You should indicate the number of visits without the use of abbreviations, besides that utilize the `|pluralize` functionality to be more descriptive here according to the example.